db/migrate/20220602234031_create_user.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUser < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :username
+      t.string :email
+      t.string :password_digest
+      t.string :role
```

Comment:
I noticed you added the role, but it was not defined in the creation of the user. I think we should add a default value to 'user' so we can differentiate it from the admin